README.md

```diff
@@ -37,6 +37,7 @@ of riscv-tools:
     $ cd rocket-chip/riscv-tools
     $ git submodule update --init --recursive
     $ export RISCV=/path/to/install/riscv/toolchain
+    $ export MAKEFLAGS="$MAKEFLAGS -jN"
```

Comment:
I could do that, and what do you think about a comment explaining the need to replace N? I noticed that there are 14 other uses of "-jN" in the document and a comment "Assuming you have N cores on your host system".